chatmeleon-server/src/middlewares/error-handler.ts

```diff
@@ -1,48 +0,0 @@
-import { Request, Response, NextFunction } from "express";
```

Comment:
I think we should still keep a general error handling middleware, just to log the error and response with a 500 Internal Error